src/config/passportConfig.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  facebookAuth: {
+    clientID: process.env.FACEBOOK_CLIENT_ID,
+    clientSecret: process.env.FACEBOOK_APP_SECRET,
+    callbackURL: `http://${process.env.HOST}:${process.env.PORT}/api/v1/auth/facebook/callback`,
```

Comment:
This will not work in production. Research on how to fix this?

Reply:
@Temmyogunbo i have implemented https for production